DataFormats/PatCandidates/src/classes_def_other.xml

```diff
@@ -1,6 +1,9 @@
 <lcgdict>
  <selection>
 
+   <class name="std::vector<std::vector<float> >" />
+   <class name="edm::Wrapper<std::vector<std::vector<float> > >" />
```

Comment:
The dictionary for this STL-based type is already available in 
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DataFormats/StdDictionaries/src/classes_def_vector.xml#L32

please remove it from this file.

the pair string-vectorFloat below is also misplaced, but I suppose it's not for you